docs/docs/concepts/foundation/state_model/storage_slots.md

```diff
@@ -28,7 +28,7 @@ For structs and arrays, we are logically using a similar storage slot computatio
 
 Private storage is a different beast. As you might remember from [State Model](./main.md), private state is stored in encrypted logs and the corresponding private state commitments in append-only tree where each leaf is a commitment. Being append-only, means that leaves are never updated or deleted; instead a nullifier is emitted to signify that some note is no longer valid. A major reason we used this tree, is that lookups at a specific storage slot would leak information in the context of private state. If you could look up a specific address balance just by looking at the storage slot, even if encrypted you would be able to see it changing! That is not good privacy. 
 
-Following this, the storage slot as we know it doesn't really exist. The leaves of the note hashes tree are just commitments to content (think of it as a hash of its content). 
+Following this, the storage slot as we know it doesn't really exist. The leaves of the note hashes tree are just commitments to content (think of it as a hash of its content). One could think "wait, then if someone encrypts a log to me with the details of a transaction, can I decrypt it before the sequencer adds its commitment to the note hash tree"? Well, yes. You cannot "un-see it" even if the transaction gets reorg'd out of the chain later, but you cannot use it because the sequencer doesn't have it in the global state. The log is part of the transaction and is revealed when it is broadcasted.
```

Comment:
I thought about that, but those are supposed to be dev docs, as in "hands-on" stuff. Thought about including them in a more "explanation" section even if it could fit somewhere else.

Do you think adding it as a note [here](https://github.com/AztecProtocol/aztec-packages/blob/3456142b898f77401656c15890bdc66566e4ec05/docs/docs/dev_docs/contracts/resources/common_patterns/main.md?plain=1#L92) would make more sense?

Reply:
Agree on not conflating this with storage slots. I think this could be a nice item in the anti-patterns list: you should not emit an encrypted log with sensitive information (eg a passphrase to access an offchain resource) that a recipient should only see if a tx succeeds, since the recipient could still inspect the encrypted tx logs from the mempool and decrypt the log meant for them.

Reply:
I agree with santiago, that is a good place to put it!